content/en/docs/reference/command-line-tools-reference/feature-gates/aggregated-discovery-endpoint.md

```diff
@@ -13,6 +13,9 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.27"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.30"
```

Comment:
Please consider adding the `toVersion` in the previous stage to align with consistency across other feature gate files.
_(Refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information)_
```suggestion
    toVersion: "1.29"
  - stage: stable
    defaultValue: true
    fromVersion: "1.30"
```